src/components/SidebarNav/SidebarNav.tsx

```diff
@@ -120,14 +120,16 @@ export const SidebarNav: React.FC<SidebarNavProps> = ({
             }}
           >
             <Popover userId={user?.id} show={isHovered} />
-            <div className="flex items-center p-2 px-4 gap-3 max-w-[224px] overflow-hidden">
+            <div className="flex items-center justify-center p-2 lg:px-4 gap-2 max-w-[224px] overflow-hidden">
               <ProfilePicture
                 className="min-w-10 min-h-10"
                 img={user?.profileImage || ''}
                 userName={user.name || ''}
               />
               {screen.width > 1024 && (
-                <Body1 className="text-white text-wrap">{user?.name}</Body1>
+                <Body1 className="text-white text-wrap max-w-20">

```

Comment:
I would remove the max-w-20
For large names like mine it seems a little tight
![image](https://github.com/sirius-valley/tricker-front/assets/87399661/f004c65c-9be1-43ca-8d55-fb4c60a93db3)
without:
![image](https://github.com/sirius-valley/tricker-front/assets/87399661/e83aba70-77e9-4f99-8e9f-6c65e050dedf)

Reply:
I added it to improve responsiveness. If you notice, when you shrink the screen, there's a point where the sidebar starts to get cut off (around 770px) and a slight horizontal scroll is added. If you find another way to improve it, I'm open to suggestions.